.github/workflows/schedule_build.yml

```diff
@@ -0,0 +1,45 @@
+# -----------------------------------------------------------------------------
+#   - invoked on push, pull_request, or manual trigger
+#   - test under at least 3 versions of python
+# -----------------------------------------------------------------------------
+name: schedule-build
+on:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
Is it possible to combine this with the build.yaml by doing something like this?
```
on:
  - push
  - pull_request
  - workflow_dispatch
  - schedule:
    - cron: "0 0 * * *"
```

Reply:
That should be possible